src/include/firebird/IdlFbInterfaces.h

```diff
@@ -1990,7 +1991,7 @@ namespace Firebird
 
 		static const int EXECUTE_FAILED = -1;
 		static const int SUCCESS_NO_INFO = -2;
-		static const unsigned NO_MORE_ERRORS = 0xffffffff;
+		static const unsigned NO_MORE_ERRORS = 0x7fffffff;
```

Comment:
I don't understand this change, it doesn't seem related to your changes inside the .idl file.

Reply:
So do I. I have no idea why it is here.

Reply:
Perhaps @asfernandes should look at it. CLOOP is his pet.

Reply:
> Perhaps @asfernandes should look at it. CLOOP is his pet.

If you have any reproducible test case sent it to Firebird developers.

If you do not like any piece of code there, you can remove and reimplement them in your side products copied from Firebird.

Reply:
How is it related to obviously wrongly generated line above?